tests/mock_network/churn.rs

```diff
@@ -723,8 +723,21 @@ impl Expectations {
         }
 
         for key in self.messages {
-            // All received messages for single nodes were removed: if any are left, they failed.
-            assert!(key.dst.is_multiple(), "Failed to receive request {:?}", key);
+            if let Authority::Node(curr_name) = key.dst {
```

Comment:
the test always only send message to `new name`s ?

Reply:
No, we send to old name, so this is buggy. Good catch!